### DISCUSSION TITLE
Security Risk?

### DISCUSSION THREAD

**Discussion by KingTut1762** [code4recovery/12-step-meeting-list#511] [Q&A] [answered]
Greetings,
Regarding the basic PDF schedule, would there be an administrative security risk if the links to the pdf's were published and accessible to visitors of our website?

**Comment by joshreisner** [comment, marked as answer]
I can't think of one, but can you be more specific about what type of risk you're thinking of?

Also are you referring to the PDF service at pdf.code4recovery.org, or in your WordPress plugin?

**Reply by KingTut1762** [reply]
Thanks for the quick response--
My colleagues were worried about giving visitors access to something that is only available to administrators which might lead to unauthorized access to our website. The issue stems from our plugin, I just took a look at your PDF service for the first time just now.
I'm a simple editor and I thought it funny to even ask because we just aren't that important (LOL), our info isn't top secret stuff...
If someone wanted to ransack our website, my guess is they wouldn't do it through a link to a PDF. 
---Thanks for looking at this and I will tell my colleagues not to hide their 4th Steps in the website---
